exasol/toolbox/tools/security.py

```diff
@@ -396,12 +298,15 @@ def create(
     { "cve": "<cve-id>", "cwe": "<cwe-id>", "description": "<multiline string>", "coordinates": "<string>", "references": ["<url>", "<url>", ...] }
 
     Output:
-    Links to the created issue(s)
+    { "cve": "<cve-id>", "cwe": "<cwe-id>", "description": "<multiline string>", "coordinates": "<string>", "references": ["<url>", "<url>", ...], "issue_url": "<url>" }
```

Comment:
I think this method `create()` and its docstring could be an excellent reference for many of the other classes, methods and functions. Please consider linking this if appropriate.